e2e/cypress/integration/local/projectDatasets.spec.ts

```diff
@@ -0,0 +1,118 @@
+/// <reference types="cypress" />
+/*!
+ * Copyright 2022 - Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import "../../support/utils";
+import Fixtures from "../../support/renkulab-fixtures";
+import "../../support/datasets/gui_commands";
+
+describe("Project dataset", () => {
+  const fixtures = new Fixtures(cy);
+  const useMockedData = Cypress.env("USE_FIXTURES") === true;
+  const projectPath = "e2e/testing-datasets";
+
+  beforeEach(() => {
+    fixtures.config().versions().userTest();
+    fixtures.projects().landingUserProjects();
+    fixtures.project(useMockedData, projectPath);
+    fixtures.projectKGDatasetList(useMockedData, projectPath);
+    fixtures.projectDatasetList(useMockedData);
+    fixtures.projectTestContents(undefined, 9);
+    fixtures.projectMigrationUpToDate("*");
```

Comment:
With the suggestions to how to change the fixtures, this would become:
```suggestion
    fixtures.projectMigrationUpToDate({repoUrl: "*"});
```